addons/website_slides/models/slide_slide.py

```diff
@@ -31,6 +31,14 @@ class SlidePartnerRelation(models.Model):
     completed = fields.Boolean('Completed')
 
 
+class SlideLink(models.Model):
+    _name = 'slide.external.link'
```

Comment:
I would call it either slide.slide.link or slide.link, but not external. What fi I want to put an internal link to my website, hein ? You are bien feinté ! :man_dancing:

Reply:
Go for `slide.slide.link` to avoid same mistake with tag, and having links on channel maybe one day with a decent name ....